system-probe_arm64/Dockerfile

```
@@ -1,8 +1,8 @@
 FROM debian:buster
 
 ARG DEBIAN_FRONTEND=noninteractive
-ARG GO_VERSION=1.20.5
-ARG GO_SHA256=aa2fab0a7da20213ff975fa7876a66d47b48351558d98851b87d1cfef4360d09
+ARG GO_VERSION=1.20.6
+ARG GO_SHA256=b945ae2bb5db01a0fb4786afde64e6fbab50b67f6fa0eb6cfa4924f16a7ff1eb
```

I think it's `4e15ab37556e979181a1a1cc60f6d796932223a0f5351d7c83768b356f84429b` here ?
```suggestion
ARG GO_SHA256=4e15ab37556e979181a1a1cc60f6d796932223a0f5351d7c83768b356f84429b
```